spec/jobs/link_discourse_account_job_spec.rb

```diff
@@ -14,29 +14,20 @@
 
   describe "#perform" do
     context "existing account" do
-      it "it sets external_id to the one found on Discourse if user is confirmed" do
-        user = FactoryBot.create(:user, external_id: nil, confirmed_at: DateTime.now)
-
-        expect_any_instance_of(DiscourseService).to receive(:find_user_by_email).and_return({"id" => 10})
-        expect_any_instance_of(DiscourseService).not_to receive(:invite_user)
-
-        perform_enqueued_jobs { LinkDiscourseAccountJob.perform_later(user) }
-
-        user.reload
-        expect(user.external_id).to eq(10)
-      end
-
-      it "it send confirmation email if user is on Discourse but not confirmed" do
+      it "it links user and sends welcome email" do
```

Comment:
What means to `link` as user? If I see the description of the Pull request I understand, but If I come to read the tests I may get lost... what about _"it links community user and with membership app and sends welcome email"_ or something like that?

Reply:
Yeah I think this can be improved